src/main/resources/application-dev.properties

```diff
@@ -2,10 +2,10 @@ spring.datasource.url=jdbc:postgresql://localhost:26257/defaultdb
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.datasource.username=root
 spring.datasource.password=
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
this should probably stay unchanged

Reply:
good point

Reply:
Changed back. Would be great to create workaround in the future. I need to use database differently sometimes and it is easy to forget to change it back to original state. Noticed same thing in other pull request and I am certain this will happen again in the future.